### DISCUSSION TITLE
Is relay endpoint still considered in beta?

### DISCUSSION THREAD

**Discussion by yairopro** [hasura/graphql-engine#7795] [Question] [answered]
Hi

Is relay endpoint still considered in beta?

**Comment by ghost** [comment, marked as answer]
TLDR: Yes, still in "beta" with no current plans to expand support :)

Hi @yairopro - Thanks so much for bringing up our relay endpoint functionality with our current postgres support.  We currently are not planning on expanding support for relay endpoints but will be keeping our ear to the ground if there is a growing need for that feature beyond its current implementation.

Please let us know if you have growing requirements in your work for relay endpoints and I can add that to our backlog prioritization tracking!

**Reply by yairopro** [reply]
thanks for your reply

**Reply by dgca** [reply]
> no current plans to expand support :)

Oof, I really hope this isn't the case. Currently, the latest Hasura docs say this about Relay:

> Currently, Hasura’s Relay schema doesn’t expose remote schemas or actions. This will be fixed in future releases.

I really hope custom actions are brought to Relay, as they are pretty much a make or break feature, IMO. Remote schemas would allow for a less elegant but still workable solution. Neither would mean that we may have to drop Hasura in the future.

Knowing the workarounds we're currently implementing to get around the existing limitations may never be supported would make it very difficult to stick to Hasura.

**Comment by 8byr0** [comment]
I opened an enh issue https://github.com/hasura/graphql-engine/issues/7950 to collect feedback and interest for relay if you want to join!
